### PR TITLE
Add test activity + layouts for assignments+awards

### PR DESCRIPTION
**Pull request by karllindmark**
@peter-budo 

I added a test activity (accessible via the MENU on debug builds) that we can use to show layouts without disrupting the other parts of the app.

I think these turned out quite nicely. My thought is that we merge these into dev, and then one PR per layout for the actual implementation.

### CODE REVIEW

**Comment by peter-budo** [comment]
@karllindmark failed build

**Comment by karllindmark** [comment]
@peter-budo: Yup, Java issues at Travis. :+1:

**Comment by karllindmark** [comment]
Build from Jenkins: http://ci.peterscorner.co.uk/job/BF4-Intel-PR/

File: BF4Intel/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -218,4 +219,8 @@
   <string name="label_sort_progress">By progress</string>
   <string name="label_completed">Completed</string>
   <string name="label_offline_mode">Offline mode</string>
+  <string name="menu_test">Test</string>
+  <string name="assignment_prerequisite">Pre-requisites</string>
+  <string name="assignment_tasks">Tasks</string>
```
Comment (peter-budo) [inline BF4Intel/src/main/res/values/strings.xml]: This should be `AWARD REQUIREMENTS`

**Comment by peter-budo** [comment]
There is lot of hard coded strings. I know they will be removed later, but this should never reach dev.
- Do you think that titles could benefit from bold?
- Could you add count of ribbons till next medal?

File: BF4Intel/src/main/res/values/strings.xml (hunk 1 above)
Comment (karllindmark) [inline BF4Intel/src/main/res/values/strings.xml]: Fixed in upcoming commit. :-)

**Comment by karllindmark** [comment]
@peter-budo: Not sure what I think of bolding (as I've been using the card look of Google Now as a base), but I've attached two screenshots; one with bold, one with blue coloring. :-)

<a href="https://cloud.githubusercontent.com/assets/979152/2811054/09582902-ce01-11e3-97c7-5804d3ef357e.png" style="float: left"></a>
<a href="https://cloud.githubusercontent.com/assets/979152/2811055/095893ce-ce01-11e3-9954-000bf895e05f.png" style="float: left"></a>

Any thoughts?

Count of ribbons is available in that second screenshot. :-)

**Comment by peter-budo** [comment]
@karllindmark you gone hate, but here it comes. I do think that battlelog layout works better.
However I would prefer title in the holo blue but would like to see it in middle. Rest of the layout preferably as battlelog

**Comment by karllindmark** [comment]
@peter-budo I'm not completely sure what you mean, but... 

... could you create a mockup of your suggestion, and post the screenshot here? That way we could easily do a comparison between the current suggestion and a possible fork of it. :-)

**Comment by peter-budo** [comment]
@karllindmark simple like that

https://www.dropbox.com/s/a9l1cc5ekbi72uo/Screenshot_2014-04-27-19-58-48.png

**Comment by karllindmark** [comment]
@peter-budo: I think that does look a bit nicer - could you commit & push your award layout changes to this PR?

**Comment by karllindmark** [comment]
I'll give this a look in the morning @peter-budo :-)

**Comment by karllindmark** [comment]
@peter-budo: Ready to be merged? :-)

**Comment by peter-budo** [comment]
@karllindmark I setup pivotal chore to clean up setup from action bar if you can please do it

**Pull request merged**